allennlp/data/tokenizers/word_stemmer.py

```diff
@@ -16,8 +17,10 @@ class WordStemmer(Registrable):
     """
     default_implementation = 'pass_through'
 
-    def stem_word(self, word: str) -> str:
-        """Converts a word to its lemma"""
+    def stem_word(self, word: Token) -> Token:
+        """
+        `Modifies` the ``text`` field in the input token, and also returns the same token.
```

Comment:
looking at the two implementations below, one of them does return the same token, one of them doesn't.

which is the correct behavior?

Reply:
Yeah, I wrote all of that before I realized that you can't modify the `text` field in spacy tokens...  I'll fix the wording.

Reply:
Done.